src/governance.cpp

```diff
@@ -619,14 +618,15 @@ bool CGovernanceManager::ProcessVote(CNode* pfrom, const CGovernanceVote& vote,
     uint256 nHashGovobj = vote.GetParentHash();
     object_m_it it = mapObjects.find(nHashGovobj);
     if(it == mapObjects.end()) {
-        mapOrphanVotes.Insert(nHashGovobj, vote);
-        RequestGovernanceObject(pfrom, nHashGovobj);
-        std::ostringstream ostr;
-        ostr << "CGovernanceManager::ProcessVote -- Unknown parent object "
-                << ", MN outpoint = " << vote.GetVinMasternode().prevout.ToStringShort()
-                << ", governance object hash = " << vote.GetParentHash().ToString() << "\n";
-        LogPrintf(ostr.str().c_str());
-        exception = CGovernanceException(ostr.str(), GOVERNANCE_EXCEPTION_WARNING);
+        if(mapOrphanVotes.Insert(nHashGovobj, vote)) {
```

Comment:
Can be slightly improved imo:
```
        std::ostringstream ostr;
        ostr << "CGovernanceManager::ProcessVote -- Unknown parent object"
             << ", MN outpoint = " << vote.GetVinMasternode().prevout.ToStringShort()
             << ", governance object hash = " << vote.GetParentHash().ToString() << "\n";
        exception = CGovernanceException(ostr.str(), GOVERNANCE_EXCEPTION_WARNING);
        if(!mapOrphanVotes.Insert(nHashGovobj, vote)) {
            LogPrint("gobject", ostr.str().c_str());
            return false;
        }
        LogPrintf(ostr.str().c_str());
        RequestGovernanceObject(pfrom, nHashGovobj);
```
This way you can 1) create exception properly in both cases 2) log error in debug mode (optional, but I guess could be helpful on mainnet)